### ISSUE TITLE
All AI data is gone, even with the Datasets activated

### ISSUE THREAD

**Issue by jjiglesiasg** [facebook/Rapid#1202]
### Description

No AI data shown

### Screenshots

![Screenshot (119)](https://github.com/facebook/Rapid/assets/22849578/6b03ebfb-499b-4746-984f-9d381a2a92d4)
![Screenshot (118)](https://github.com/facebook/Rapid/assets/22849578/b2c1ffb5-34a7-4980-965c-f11aa8b3f6ab)

### Version

2.1.0

### What browser are you seeing the problem on? What version are you running?

Chrome Version 105.0.5195.127 (Official Build) (64-bit)

### Steps to reproduce

Open any where with Rapid, No AI data shown

### The browser URL at the time you encountered the bug

https://rapideditor.org/edit#background=Bing&datasets=fbRoads,msBuildings&disable_features=boundaries&map=18.04/-16.74390/-68.80678&poweruser=true

**Comment by SoNick** [comment]
Definitely seeing the same thing here. Tested in Chrome, Firefox, and Edge all on Win10 64-bit. I think this may be related to Issue #1201

**Comment by pkoby** [comment]
Same (Linux, Firefox) and I'll add that I think this is causing a bug in the JOSM plugin. Relevant stack trace lines include:
`java.io.IOException: java.util.concurrent.ExecutionException: java.lang.RuntimeException: org.openstreetmap.josm.io.OsmTransferException: Could not connect to the OSM server. Please check your internet connection.
	at org.openstreetmap.josm.plugins.mapwithai.backend.DownloadMapWithAITask$DownloadTask.realRun(DownloadMapWithAITask.java:183)`

So maybe it's a connection issue between MapWithAI and OSM?

**Comment by jjiglesiasg** [comment]
![Screenshot (120)](https://github.com/facebook/Rapid/assets/22849578/b42238b6-9037-45a9-923e-8bfd4995da14)

Yes it was working a day ago

**Comment by misibacsi** [comment]
I also wish to report that MapwithAI does not work now. I noticed it stopped working yesterday.

**Comment by arch0345** [comment]
It looks like this and #1203 was caused by the SSL certificate for mapwith.ai expiring

**Comment by misibacsi** [comment]
I did not receive such error message about SSL. 
One problem is: no error message at all.

Questions: 
1. How can I check if MapwithAI is loaded?
2. How can I force MapwithAI to load?

**Comment by arch0345** [comment]
> I did not receive such error message about SSL. One problem is: no error message at all.

Did you try opening up DevTools? This is the error I found when attempting to load Facebook roads/Microsoft buildings:
![image](https://github.com/facebook/Rapid/assets/85302075/646fa1a5-f166-4152-a2a7-0f8151553e63)

**Comment by misibacsi** [comment]
I was just about to report to the admins of MapwithAI to update the security certificate, but it seems someone already did it.
The error is vanished. 
I can edit the OSM map again.

I guess this ticket can be closed.

**Comment by tsmock** [comment]
> Same (Linux, Firefox) and I'll add that I think this is causing a bug in the JOSM plugin.

I've updated the URL for the JOSM plugin.

Steps to get the the updated URL:
1. JOSM Preferences
2. MapwithAI preferences
3. Refresh icon next to the map
4. If the entry for MapWithAI in Selected entries does not update, remove and readd it.

Both the JOSM plugin and Rapid used [mapwith.ai](http://mapwith.ai/). I did some testing, and [rapideditor.org](http://rapideditor.org/) has access to the /maps/ml_roads endpoint, which is where the buildings and roads come from. And the Esri conflated data. (edited)

@misibacsi : You _may_ have accepted the current certificate temporarily. It is still an issue.
